shaptools/hana.py

```diff
@@ -356,7 +356,7 @@ def check_user_key(self, key):
             return False
 
     def create_user_key(
-            self, key, environment, user, user_password, database=None):
+            self, key, environment, key_user, key_password, database=None):
```

Comment:
@arbulu89 Conceptually, the key stores the user name, password and environment in a safe way. To access the DB, or you use the key (-U option), or you need to inform all the parameters (-u, -p). So, here, I think to improve the readability, the parameters could be:
```suggestion
            self, key_name, environment, user_name, user_password, database=None):
```